backends/xdma/src/XdmaBackend.cc

```diff
@@ -3,6 +3,8 @@
 
 #include "XdmaBackend.h"
 
+#include "Utilities.h"
+
```

Comment:
Reviewing the code myself once more shows me that these lines need to be removed...